File: QvAbu.Api/QvAbu.Api/Models/Questions/TextQuestion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace QvAbu.Api.Models.Questions
{
    public class TextQuestion : Question
    {
        #region Properties

        public Guid AnswerId { get; set; }

        public virtual TextAnswer Answer { get; set; }

        #endregion
    }
}

File: QvAbu.Web/src/app/models/questions/text-question.ts
import { Question, QuestionType } from './question';
import { TextAnswer } from './text-answer';
import { Guid } from '../guid';

export class TextQuestion extends Question {
  answer: TextAnswer;

  constructor(id: Guid, revision: number) {
    super();
    this.id = id;
    this.revision = revision;

    this.type = QuestionType.textQuestion;
  }
}
